src/style.scss

```diff
@@ -321,6 +321,12 @@
   }
 }
 
+:host([showOverflow]){
```

Comment:
This new attribute basically does the same thing that `:host([noanimation])` does? The removal of `overflow: hidden` breaks the animation of the accordion and that's why we named it `noanimation`. 

When using `noanimation` this does not address the issue you are having with the z-indexing and the height of the accordion when open?

Reply:
noanimation sets overflow to auto, which evaluates to hidden in our app. changing that to overflow: visible resolves the issue.

Reply:
The issue I have here is that we created `noanimation` to solve an issue reported by this team about the secondary exposed content within an according not being exposed. 

So, if `noanimation` is not solving the issue, the proper update is to update `noanimation` versus creating another API that essentially does the same thing.

Reply:
Ah, if that's the case, I'll amend the PR for that! I didn't see that it was a response to us, only looked at the code and tried to integrate with that... reading2hard.

Reply:
Also Dale...I tested locally and turned off the animation adjustment...it doesn't seem like the original animation is causing any problems:

https://user-images.githubusercontent.com/25255197/142520690-c648bd17-56af-4f3b-8c26-185d2a4d17e6.mov